tests/Tokenizer/TokenTest.php

```diff
@@ -42,9 +42,6 @@ public function testConstructorValidation($input): void
     public function provideConstructorValidationCases(): array
     {
         return [
-            [null],
```

Comment:
as method prototype allows us to provide different types, we should check what happen if one would do it

Reply:
We wanted to trust PHPDoc.

Reply:
not according to https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5900#discussion_r702286531

Reply:
I wrote "wanted", not "want", that's why it is reverted.